automata/pda/dpda.py

```diff
@@ -151,6 +162,31 @@ def _get_next_configuration(self, old_config: PDAConfiguration) -> PDAConfigurat
         )
         return new_config
 
+    def _get_input_path(
+        self, input_str: str
+    ) -> Tuple[List[Tuple[PDAConfiguration, PDAConfiguration]], bool]:
+        """
+        Calculate the path taken by input.
+
+        Args:
+            input_str (str): The input string to run on the DPDA.
+
+        Returns:
+            tuple[list[tuple[PDAConfiguration, PDAConfiguration], bool]]: A list
```

Comment:
@Vipul-Cariappa It's a small thing, but could we please match the casing of the actual type here? Also, I think the brackets are mismatched compared to the actual annotation.

```suggestion
            Tuple[List[Tuple[PDAConfiguration, PDAConfiguration]], bool]: A list
```